docs/src/man/comparisons.md

```diff
@@ -2,76 +2,162 @@
 
 This section compares DataFrames.jl with other data manipulation frameworks.
 
+A sample data set can be created using the following code:
+
+```julia
+df = DataFrame(id = 'a':'f', grp = [1,2,1,2,1,2], x = 6:-1:1, y = 4:9, z = 3:8)
+```
+
+## Comparison with the Python package pandas
+
+The following table compares the main functions of DataFrames.jl with the Python package pandas (version 1.1.0):
+
+```python
+df = pd.DataFrame(
+        {'grp': [1,2,1,2,1,2], 'x': range(6,0,-1), 'y': range(4,10), 'z': range(3,9)},
+        index = list('abcdef'))
+```
+
+Note that pandas allows row indexing by labels. For DataFrames.jl, you can achieve the same result via row filtering as shown below.
+
+How to access data:
+
+| Operation                  | Example                     | pandas                 | DataFrames.jl                                              |
```

Comment:
Why do you add "Example" column only here?

Reply:
I want to see if you like it :-)  I thought it's easier to understand the sample code. I can add the same column for other tables including dplyr/stata later today.

Reply:
Let us wait for @nalimilan to comment on this. I was afraid that for transformations the descriptions would be overly long.

Reply:
I just generated the docs and checked from a browser.  It doesn't look at great because the current style limits the width of the content and so the text starts wrapping. Perhaps it's not a good idea after all. Here's how they look:

1) With the Example column

![image](https://user-images.githubusercontent.com/1159782/91144570-14044480-e669-11ea-9924-e72210f6afaf.png)

2) Without the Example column

![image](https://user-images.githubusercontent.com/1159782/91144588-1a92bc00-e669-11ea-9498-fd3ed25ffd2b.png)

Reply:
Yeah, making the table too wide isn't great. I think we can assume people understand the pandas code, otherwise they shouldn't read the table. :-)